core/src/main/resources/db/cgds.sql

```diff
@@ -279,15 +278,31 @@ CREATE TABLE `genetic_profile_samples` (
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
 );
 
+CREATE TABLE `gene_panel` (
+    `INTERNAL_ID` int(11) NOT NULL auto_increment,
+    `STABLE_ID` varchar(255) NOT NULL,
+    `DESCRIPTION` mediumtext,
+    PRIMARY KEY (`INTERNAL_ID`),
+    UNIQUE (`STABLE_ID`)
+);
+CREATE TABLE `gene_panel_list` (
+    `INTERNAL_ID` int(11) NOT NULL,
+    `GENE_ID` int(255) NOT NULL,
+    PRIMARY KEY (`INTERNAL_ID`, `GENE_ID`),
+    FOREIGN KEY (`INTERNAL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE,
+    FOREIGN KEY (`GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
+);
 --
 -- Table structure for table `sample_profile`
 --
 CREATE TABLE `sample_profile` (
   `SAMPLE_ID` int(11) NOT NULL,
   `GENETIC_PROFILE_ID` int(11) NOT NULL,
+  `PANEL_ID` int(11) DEFAULT NULL,
   UNIQUE KEY `UQ_SAMPLE_ID_GENETIC_PROFILE_ID` (`SAMPLE_ID`,`GENETIC_PROFILE_ID`), -- Constraint to allow each sample only once in each profile
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE,
-  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
+  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE,
+  FOREIGN KEY (`PANEL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE
```

Comment:
@fedde-s we talked about your point of ON DELETE ... clauses.  It started a good conversation about what cases may come up where we want to alter or remove a previously existing gene panel. Changes like this would happen during the import / db update process.

After talking about it a bit we think that we prefer to prohibit all cascades --- so that if any profile/sample is linked to a gene panel identifier, then the gene panel cannot be deleted from the gene_panel table. First the importer/updater will need to resolve "properly" (whatever that means in the case) the old references to the gene panel identifier.

So we think we will remove the entire cascade clause at this time, and talk more about what should happen if a gene panel needs to change and how to support it in the import code.